fhir-model/src/main/java/com/ibm/fhir/model/visitor/ResourceFingerprintVisitor.java

```diff
@@ -188,11 +186,6 @@ protected void updateDigest(String name, String value) {
      * @return
      */
     protected boolean includePath() {
-        String idName = currentResourceName + ".id";
-        String versionIdName = currentResourceName + ".meta.versionId";
-        String lastUpdatedName = currentResourceName + ".meta.lastUpdated";
-        String path = getPath();
-        return !path.startsWith(idName) && !path.startsWith(versionIdName) && !path.startsWith(lastUpdatedName);
-
+        return !excludePaths.contains(getPath());
```

Comment:
Minor point that being protected, includePaths() could be called before exludePaths is initialized, resulting in a NPE. But it's not really a valid use-case, so not sure it's worth addressing.

Reply:
```suggestion
        return excludePaths == null ? null : !excludePaths.contains(getPath());
```